inst/components/scss/grid.scss

```diff
@@ -1,7 +1,14 @@
 .bslib-grid {
+  --_fallback-item-span: unset;
+
+  > * {
+    grid-column: auto/span var(--_fallback-item-span, 6);
+  }
```

Comment:
I need another refresher on what this change does for us. In particular,

* Why is the `unset` better than `1`?
* Why the "default" value of `6`?
* Is this "pseudo-private" CSS variable worth the added complexity? If we don't see it evolving into a public API thing, I'd probably prefer not having it all?

Reply:
Ahh, sorry for the premature comment, I'll come back to this after I get through the TS

Reply:
The biggest thing being solved by this code is a situation like `col_widths = breakpoints(lg = c(2, 4, 6))`, where between `sm` and `lg` the items would all have column-width of 1. (And that would not be good.)